src/Vlingo.Actors/Vlingo.Actors.csproj

```diff
@@ -1,27 +1,27 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-
-    <!-- NuGet Metadata -->
-    <IsPackable>true</IsPackable>
-    <PackageVersion>0.2.0</PackageVersion>
-    <PackageId>Vlingo.Actors</PackageId>
-    <Authors>Vlingo</Authors>
-    <Description>
-      Type safe Actor Model toolkit for reactive concurrency and resiliency using C# and other .NET languages.
-    </Description>
-    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-    <PackageLicenseUrl>https://mozilla.org/MPL/2.0/</PackageLicenseUrl>
-    <PackageProjectUrl>https://github.com/vlingo-net/vlingo-net-actors</PackageProjectUrl>
-    <PackageIconUrl>https://raw.githubusercontent.com/vlingo-net/vlingo-net-common/master/vlingo-64x64.png</PackageIconUrl>
-    <RepositoryUrl>https://github.com/vlingo-net/vlingo-net-actors</RepositoryUrl>
-  </PropertyGroup>
-  <ItemGroup>
-    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
-      <_Parameter1>$(MSBuildProjectName).Tests</_Parameter1>
-    </AssemblyAttribute>
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="Vlingo.Common" Version="0.2.0" />
-  </ItemGroup>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+
+    <!-- NuGet Metadata -->
+    <IsPackable>true</IsPackable>
+    <PackageVersion>0.2.0</PackageVersion>
```

Comment:
if you are planning to get a newer version of Actors to consume in another project, you have to modify this version to something greater. I recommend `0.2.1` for this change.